crates/turbopack-core/src/resolve/plugin.rs

```diff
@@ -0,0 +1,59 @@
+use anyhow::Result;
+use turbo_tasks::primitives::BoolVc;
+use turbo_tasks_fs::{glob::GlobVc, FileSystemPathVc};
+
+use crate::resolve::{parse::RequestVc, ResolveResultVc};
+
+#[turbo_tasks::value]
+pub struct ResolvePluginCondition {
```

Comment:
Please add comments to these public items.

Reply:
done